### DISCUSSION TITLE
Release compiled binaries for ghpc

### DISCUSSION THREAD

**Discussion by casassg** [GoogleCloudPlatform/cluster-toolkit#2434] [Ideas and feature requests]
I was wondering if it could be possible to release the go compiled binaries for ghpc as well in each release. This would reduce the overhead to run this on local machines (or on CI).

**Comment by cdunbar13** [comment]
Hello, I just wanted to confirm that we've received this message and will discuss it.  We should have a response in a few days.  Thank you!

**Comment by cdunbar13** [comment]
We discussed and while we are considering it, releasing the binaries with the external dependencies requires legal review within Google.  We will start this process, but there is no exact timeline for that right now.

**Reply by casassg** [reply]
Hi @cdunbar13 ty, btw when I say binaries I mean attaching the `make` binary in https://github.com/GoogleCloudPlatform/hpc-toolkit/releases/tag/v1.31.1 Otherwise lmk
